.github/workflows/labeler.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
```

Comment:
Esta es la definición del workflow que pone las etiquetas, como el trigger es [pull_request_target](https://docs.github.com/es/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target) no se lanzará hasta que esté en la rama main